planner/core/planbuilder.go

```diff
@@ -1500,6 +1500,17 @@ func (b *PlanBuilder) buildDDL(node ast.DDLNode) (Plan, error) {
 			table:     v.Table.Name.L,
 			err:       nil,
 		})
+		b.visitInfo = append(b.visitInfo, visitInfo{
```

Comment:
Should I add check to judge if rename in `alter table`?

Reply:
Yes, you should add these check only if the alter type is rename.

Reply:
Actually, rename needs drop_pri on old table.

Reply:
Okay, I only add check in `rename to`.
I will also add it in `alter table`.